Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.6-alpine
+
+COPY requirements.txt /app/
+
+RUN true \
+	&& pip install gunicorn \
+	&& pip install -r /app/requirements.txt \
+	&& true
+
```

Comment:
it can and very much should be copied, otherwise this Dockerfile is useless.
the mounst will overwrite the content inside the image